navigation/create_control/launch/create_control.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+
+  <!-- Load controller configuration from YAML file to parameter server -->
+  <arg name="real_robot" value="$(optenv REAL_ROBOT false)"/>
+  <node name="robot_hw_interface" pkg="create_control" type="create_control_node" if="$(arg real_robot)"
```

Comment:
I don't have a clear solution in mind but I would try to avoid `REAL_ROBOT`. Maybe since you need to call `minimal.launch`, this launch file can set it directly.

Reply:
Still pending, I'll try to fix real robot stuff if I get to test it on one...